README.md

```diff
@@ -16,6 +16,14 @@ I recommend creating a separate package inside your project to serve as the cont
 
 For an example of how to use the resulting package, check out `example/example.go`. You can also see the API it generates at [godoc.org](https://godoc.org/github.com/bouk/staticfiles/files).
 
+## Installation
+
+Install with
+
+```
+go get github.com/bouk/staticfiles
```

Comment:
The reason I recommend `-u` by default is because it leads to a much more predictable outcome.

When a beginner runs `go get -u your/package` and then has a problem, you can be certain that they will have the latest version of your package, and all of its dependencies, at the time they ran the command. So if there's an issue, it's likely in the code.

If the beginner runs `go get your/package`, the outcome is completely unpredictable. Depending on the contents of their GOPATH workspace, they may either get the latest version of your package, or an old one (if they happened to have it installed earlier, directly or indirectly). They may have the latest versions of dependencies, or old ones. And mismatched (some old, some new).

So when they have a problem, you have no idea if it's a legitimate issue with your code, or the fact that they have an outdated version of something.

That's why I think having `-u` is a safer option. But this is simply my recommendation, you should consider my arguments and your preferences and decide what's better for your project.